src/Resources/ResourceManager/Implementation/Policy/NewAzurePolicyExemption.cs

```diff
@@ -0,0 +1,167 @@
+    using Policy;
+
+    /// <summary>
+    /// Creates a policy exemption.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "PolicyExemption", DefaultParameterSetName = PolicyCmdletBase.DefaultParameterSet), OutputType(typeof(PsPolicyExemption))]
+    public class NewAzurePolicyExemptionCmdlet : PolicyCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the policy exemption name parameter.
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyExemptionNameHelp)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy exemption scope parameter
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyExemptionScopeHelp)]
+        [ValidateNotNullOrEmpty]
+        public string Scope { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy exemption display name parameter
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyExemptionDisplayNameHelp)]
+        [ValidateNotNullOrEmpty]
+        public string DisplayName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy exemption description parameter
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyExemptionDescriptionHelp)]
+        [ValidateNotNullOrEmpty]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy exemption category
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyExemptionCategoryHelp)]
+        [ValidateSet(PolicyExemptionCategory.Waiver, PolicyExemptionCategory.Mitigated)]
+        public string ExemptionCategory { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy exemption policy assignment Id.
+        /// </summary>
+        [Parameter(ParameterSetName = PolicyCmdletBase.DefaultParameterSet, Mandatory = true, ValueFromPipeline = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyExemptionPolicyAssignmentIdHelp)]
+        public PsPolicyAssignment PolicyAssignment { get; set; }
+
+        /// <summary>
+        /// Gets or sets the policy definition reference ID list when the associated policy assignment is for a policy set (initiative).
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyExemptionPolicyDefinitionReferenceIdsHelp)]
+        [ValidateNotNullOrEmpty]
+        public string[] PolicyDefinitionReferenceIds { get; set; }
```

Comment:
1.Does not support `ShouldProcess` but the cmdlet verb New indicates that it should.
1. Parameter `PolicyDefinitionReferenceIds` of cmdlet `New-AzPolicyExemption` does not follow the enforced naming convention of using a singular noun for a parameter name.

Reply:
1. Added ShouldProcess support
2. PolicyDefinitionReferenceIds is a policy term. It cannot be changed. Here is a document reference. https://docs.microsoft.com/en-us/azure/governance/policy/concepts/initiative-definition-structure#policy-definition-properties